FHEM/14_SD_WS.pm

```diff
@@ -1471,6 +1473,21 @@ sub SD_WS_Parse {
                                 }
                               },
     },
+    124 => {
+        # WH31 
+        sensortype => 'WH31',
+        model      => 'SD_WS_124',
+        prematch   => sub { ($rawData,undef) = @_; return 1 if ($rawData =~ /^(30|37)/); },
+        id         => sub {my (undef,$bitData) = @_; return (SD_WS_binaryToNumber($bitData,17,19) + 1);},
```

Comment:
Mir wäre es lieb, wenn wir die Id in hex übernehmen. So ist es auch bei WH51 und WH 57.

```
        id         => sub {my ($rawData,undef) = @_; return return substr($rawData,2,2); },
```

Reply:
Ja das wäre besser, gebe ich dir recht